tests/integration_tests/resources/dsl/scripts/workflows/context.py

```diff
@@ -0,0 +1,6 @@
+from cloudify.workflows import ctx
```

Comment:
```suggestion
from cloudify import ctx
```
we can't use the workflow context here, this needs to be operation ("regular") context. This is in an operation, not in a workflow
`from cloudify.state import ctx` also works.

This should really error out in a nicer way, I've made RD-4209 about it